edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Service/DependencyManager.cs

```diff
@@ -95,11 +95,15 @@ public void Register(ContainerBuilder builder)
             this.RegisterMqttModule(builder, storeAndForward, optimizeForPerformance);
             this.RegisterAmqpModule(builder);
             builder.RegisterModule(new HttpModule(this.iotHubHostname));
+
+            if (experimentalFeaturesConfig.GetValue(Constants.ConfigKey.MqttBrokerEnabled, false))

```

Comment:
We should probably use the ExperimentalFeatures type? https://github.com/Azure/iotedge/blob/master/edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Service/ExperimentalFeatures.cs